tests/fixtures/webextension_es6_module/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "applications": {
+    "gecko": {
+      "id": "webExtTest@asamuzak.jp"
```

Comment:
Ant reason why this isn’t `@webextension-guid` like every other current test fixture?

Reply:
Fixed, I think I simply didn't change it from the example that I fetched from one of the issues.